templates/spa.html

```diff
@@ -44,7 +43,7 @@
   <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
   <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@300;400;500&amp;display=swap" rel="stylesheet">
 
-  <style>{% inline_file "/static/css/main.css" %}</style>
+  <link rel="stylesheet" href="/static/css/main.css">
```

Comment:
Let's add `?v={{app_version}}` to each of these URLs.  

We use that in existing client-side includes to cause the client to load the newest version of these assets whenever we deploy a new version to GAE.